minifi_main/AgentDocs.cpp

```diff
@@ -43,27 +81,17 @@ void AgentDocs::generate(const std::filesystem::path& docsdir, std::ostream &gen
   std::map<std::string, ClassDescription> processorSet;
   for (const auto &group : minifi::AgentBuild::getExtensions()) {
     struct Components descriptions = build_description_.getClassDescriptions(group);
-    for (const auto &processorName : descriptions.processors_) {
-      processorSet.insert(std::make_pair(extractClassName(processorName.full_name_), processorName));
+    for (const auto& processor_description : descriptions.processors_) {
+      processorSet.insert(std::make_pair(extractClassName(processor_description.full_name_), processor_description));
     }
   }
   for (const auto &processor : processorSet) {
     const auto& filename = docsdir / processor.first;
```

Comment:
I think we could also add the .md extension to these files

Reply:
We could, but then we'd need to remove the extension later, at line 171.  I think it would be better to do this separately, together with either removing or fixing the logic of the `*.extra` files.